.storybook/main.ts

```diff
@@ -3,7 +3,11 @@ import type { StorybookConfig } from '@storybook/nextjs';
 
 const config: StorybookConfig = {
   stories: ['../components/**/*.stories.tsx'],
-  addons: ['@storybook/addon-controls', '@storybook/addon-interactions'],
+  addons: [
+    '@storybook/addon-controls',
+    '@storybook/addon-interactions',
+    '@storybook/addon-styling',
```

Comment:
What is this addon, and why is it needed?

Reply:
IDK it's on storybook docs for tailwind x'D

Reply:
Don't add things you don't know why they need to be added. This is likely needed for Tailwind integration.

This seems only to add a "dark" and "light" mode toggle, but we won't use this, as our Next.js ThemeProvider has its logic for Dark/Light mode theme.

Please remove this package, as it's even deprecated.

![image](https://github.com/nodejs/nodejs.org/assets/12037269/573cb205-163c-4fc3-ad22-5c91b66772b6)

My advice: Think before doing things.